crates/ruff_python_parser/src/snapshots/ruff_python_parser__parser__tests__with_statement.snap

```diff
@@ -1179,4 +1179,108 @@ expression: "parse_suite(source, \"<test>\").unwrap()"
             ],
         },
     ),
+    With(
+        StmtWith {
+            range: 560..581,
+            is_async: false,
+            items: [
+                WithItem {
+                    range: 566..569,
```

Comment:
This is key: the length of the `WithItem` is 3, while the length of the inner item is 1.